Controllers/UsersControllers.js

```diff
@@ -0,0 +1,18 @@
+// Requring The UserSchema
+const User = require('../Models/userSchema');
+
+// Users Registration
+module.exports.CreatingUsers = (req,res,next) => {
```

Comment:
Names shouldn't start with capital letter. It should be `createUser`.

Reply:
Resolved